gateway/ln-gateway/src/lib.rs

```diff
@@ -252,6 +252,14 @@ impl Gateway {
             GatewayError::Other(anyhow::anyhow!("Invalid federation member string {}", e))
         })?;
 
+        for actor in self.actors.lock().await.values() {
+            if actor.read().await.get_info()?.federation_id == connect.id {
```

Comment:
@natto1784 I guess that using `select_actor` would make it simpler, wdyt ?

Reply:
indeed. my bad, it was right there too